src/iap.ts

```diff
@@ -178,15 +136,13 @@ export const getProducts = <SkuType extends string>(
 ): Promise<Array<Product<SkuType>>> =>
   (
     Platform.select({
-      ios: async () => {
-        if (!RNIapIos) return [];
-
-        return RNIapIos.getItems(skus).then((items: Product[]) =>
+      ios: getIosModule()
+        .getItems(skus)
+        .then((items: Product[]) =>
           items.filter((item: Product) =>
             skus.includes(item.productId as SkuType),
           ),
-        );
-      },
+        ),
       android: async () => {
         const products = await getAndroidModule().getItemsByType(
           ANDROID_ITEM_TYPE_IAP,
```

Comment:
The issue has been raised in the current lines of code #1454. However, I see no differences.
I'll look into it.

@mathmania @andresesfm